app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -169,6 +185,35 @@ public String convertMarkup(String markup, Context context, boolean isExportInLi
             head += CSS_PRESENTATION_BEAMER;
         }
 
+        // Extract YAML Front Matter
+        if (!enablePresentationBeamer) {
+            if (appSettings.isMarkdownYamlDisplayEnabled() && markup.startsWith("---")) {
+                Map<String, List<String>> yamlFrontMatterMap = Collections.EMPTY_MAP;
+                yamlFrontMatterMap = extractYamlFrontMatter(markup);
+
+                if (!yamlFrontMatterMap.isEmpty()) {
+                    for (Map.Entry<String, List<String>> entry : yamlFrontMatterMap.entrySet()) {
+                        String key = entry.getKey();
+                        List<String> valueList = entry.getValue();
+                        String value = "";
+
+                        if (key.equals("tags")) {
```

Comment:
I'd say it's fine to leave sorting up to the user. if they add "math" as first tag before "chemistry", they probably do it intentionally so math appears more on top. if not they still can reorder on own.

thats basically also what I do on my jekyll blog posts - most important tags and categories are at top :smile: .

Reply:
fixed